code/processes/wdb.q

```diff
@@ -226,7 +226,7 @@ d:()!()
 doreload:{[pt]
 	.wdb.reloadcomplete:0b;
 	/-inform gateway of reload start
-	informgateway["reloadstart[]"];
+	informgateway[(`reloadstart;[])];
```

Comment:
I'm not sure if [] works here, but it's not the standard way to do it.  Can you change to

(`reloadstart;`)